webviz_subsurface/plugins/_reservoir_simulation_timeseries_onebyone.py

```diff
@@ -336,13 +388,18 @@ def _render_date(ensemble, clickdata, vector):
                 # json.dumps(f"{date}"),
                 table_rows,
                 table_columns,
+                (
+                    f"{simulation_vector_description(vector)} ({vector})"
+                    if get_unit(self.smry_meta, vector) == ""
+                    else f"{simulation_vector_description(vector)} ({vector})"
+                    + f" [{get_unit(self.smry_meta, vector)}]"
+                ),
```

Comment:
```suggestion
                f"{simulation_vector_description(vector)} ({vector})" + ("" if get_unit(self.smry_meta, vector) == "" else f" [{get_unit(self.smry_meta, vector)}]")
```

Reply:
👍  will change this dependent on the conclusion on `get_unit()`